tests/test_density_baselines.py

```diff
@@ -78,26 +79,22 @@ def test_density_reward(density_type, is_stationary):
 
 
 @pytest.mark.expensive
-@parametrize_density_stationary
-def test_density_trainer(density_type, is_stationary):
+def test_density_trainer_smoke():
+    # tests whether density trainer runs, not whether it's good
+    # (it's actually really poor)
     env_name = "Pendulum-v0"
     rollout_path = "tests/data/expert_models/pendulum_0/rollouts/final.pkl"
-    rollouts = types.load(rollout_path)
+    rollouts = types.load(rollout_path)[:8]
     env = util.make_vec_env(env_name, 2)
     imitation_trainer = util.init_rl(env)
     density_trainer = DensityTrainer(
         env,
         rollouts=rollouts,
         imitation_trainer=imitation_trainer,
-        density_type=density_type,
-        is_stationary=is_stationary,
+        density_type=STATE_ACTION_DENSITY,
+        is_stationary=False,
         kernel="gaussian",
     )
-    novice_stats = density_trainer.test_policy()
-    density_trainer.train_policy(2000)
-    good_stats = density_trainer.test_policy()
-    # Novice is bad
-    assert novice_stats["return_mean"] < -500
-    # Density is also pretty bad, but shouldn't make things more than 50% worse.
-    # It would be nice to have a less flaky/more meaningful test here.
-    assert good_stats["return_mean"] > 1.5 * novice_stats["return_mean"]
+    # novice_stats = density_trainer.test_policy()
```

Comment:
work in progress?

Reply:
The previous test didn't actually do anything—`novice_stats["return_mean"]` was negative, so the previous comparison only checked whether the trained policy was no more than 50% worse (!) than the original. I decided to convert this to a smoke test because the density baselines are too poor to construct a reliable test for (the basic approach is not well-justified theoretically, and only works in practice for a limited set of environments).

Reply:
I left the `novice_stats`/`good_stats` comments in just in case somebody wants to try their hand at making a non-flaky test in the future.

Reply:
OK, I'd lean towards removing these lines, but if you want to keep them then can you add a comment explaining why they're there? If I come across commented out code in a codebase with no explanation it's a bit disconcerting.

Reply:
Will remove